docs/bin/plugin_formatter.py

```diff
@@ -293,8 +293,14 @@ def get_plugin_info(module_dir, limit_to=None, verbose=False):
         # use ansible core library to parse out doc metadata YAML and plaintext examples
         doc, examples, returndocs, metadata = plugin_docs.get_docstring(module_path, fragment_loader, verbose=verbose)
 
-        if metadata and 'removed' in metadata.get('status'):
-            continue
+        if metadata:
+            metadata_status = metadata.get('status', [])
+            if not metadata_status:
+                display.error("*** ERROR: ANSIBLE_METADATA.status is empty")
```

Comment:
display.error already prepends an `[ERROR]` , so it is redundant to add more

Reply:
In the end, unsure this should cause an error, the following would fix and continue working:
`if metadata and 'removed' in metadata.get('status', []):`

Reply:
I thought about that before, but then you won't get error message about this. So separated it by two ifs. I am OK to merge them.

Reply:
that is the question, SHOULD this be an error?